doc/smart-switch/pmon/smartswitch-pmon.md

```diff
@@ -436,34 +429,51 @@ is_midplane_reachable(self):
 #### 3.1.5 ModuleBase class new APIs
 
 ##### 3.1.5.1 Need for consistent storage and access of DPU reboot cause, state and health
+#### Reboot Cause
 1.  The smartswitch needs to know the reboot cause for DPUs. Please refer to the CLI section for the various options and their effects when executed on the switch and DPUs. 
 
 * Each DPU will update its reboot cause history in the Switch ChasissStateDB upon boot up. The recent reboot-cause can be derived from that list of reboot-causes.
 * The get_reboot_cause will return the current reboot-cause of the module.
 * For persistent storage of the DPU reboot-cause and reboot-caue-history files use the existing host storage path and mechanism.
 
-#### Schema for REBOOT_CAUSE - switch stateDB
+#### Schema for REBOOT_CAUSE of SWITCH on switch stateDB
 ```
   Key: "REBOOT_CAUSE|2023_06_18_14_56_12"
 
   "REBOOT_CAUSE|2023_06_18_14_56_12": {
     "value": {
-      "cause": "REBOOT_CAUSE_HOST_RESET_DPU",
+      "cause": "Unknown",
       "comment": "N/A",
-      "device": "DPU5",
       "time": "2023_06_18_14_56_12",
       "user": "N/A"
     }
-  },
+  }
 
 ```
+#### Schema for REBOOT_CAUSE of DPUs on switch ChassisStateDB
+```
+  Key: "REBOOT_CAUSE|DPU0|2024_06_06_09_31_18"
+
+  "REBOOT_CAUSE|DPU0|2024_06_06_09_31_18": {
+    "value": {
+      "cause": "Software causes (Reboot)",
+      "comment": "User issued 'reboot' command [User: admin, Time: Thu Jun  6 09:46:43 AM UTC 2024]",
+      "device": "DPU0",
+      "time": "N/A",
+      "user": "N/A"
+    }
+  }
+
+```
+#### DPU State
```

Comment:
@rameshraghupathy can you please specify that this update is done by Chassisd inside PMON. We don't need DPU specific agent to fetch these ?

Reply:
> @rameshraghupathy can you please specify that this update is done by Chassisd inside PMON. We don't need DPU specific agent to fetch these ?

Updated:
 Store the state progression (dpu_midplane_link_state, dpu_control_plane_state, dpu_data_plane_state) on the host ChassisStateDB using the push model specified in [section: 3.2.4 of SONiC Chassis Platform Management & Monitoring HLD](https://github.com/sonic-net/SONiC/blob/master/doc/pmon/pmon-chassis-design.md)
@prgeor